File: CPP/Bioskop.cpp
using namespace std;

class Bioskop {
    private:
        int id;
        string name;
        string address;
        int totalStudios;
        float rating;

    public:
        Bioskop() {
            id = 0;
            name = "";
            address = "";
            totalStudios = 0;
            rating = 0.00;
        }

        Bioskop(int id, string name, string address, int totalStudios, float rating) {
            this->id = id;
            this->name = name;
            this->address = address;
            this->totalStudios = totalStudios;
            this->rating = rating;
        }

        // Setter and Getter for id
        void setId(int id) {
            this->id = id;
        }
        int getId() {
            return id;
        }

        // Setter and Getter for name
        void setName(string name) {
            this->name = name;
        }
        string getName() {
            return name;
        }

        // Setter and Getter for address
        void setAddress(string address) {
            this->address = address;
        }
        string getAddress() {
            return address;
        }

        // Setter and Getter for totalStudios
        void setTotalStudios(int totalStudios) {
            this->totalStudios = totalStudios;
        }
        int getTotalStudios() {
            return totalStudios;
        }

        // Setter and Getter for rating
        void setRating(float rating) {
            this->rating = rating;
        }
        float getRating() {
            return rating;
        }

        ~Bioskop() {}
};

File: Python/Bioskop.py
class Bioskop:
    # constructor
    def __init__(self, id:int, name:str, address:str, totalStudios:int, rating:float):
        self.__id = id
        self.__name = name
        self.__address = address
        self.__totalStudios = totalStudios
        self.__rating = rating

    # Setter and Getter for ID
    def setId(self, id): self.__id = id
    def getId(self): return self.__id

    # Setter and Getter for name
    def setName(self, name): self.__name = name
    def getName(self): return self.__name

    # Setter and Getter for address
    def setAddress(self, address): self.__address = address
    def getAddress(self): return self.__address

    # Setter and Getter for totalStudios
    def setTotalStudios(self, totalStudios): self.__totalStudios = totalStudios
    def getTotalStudios(self): return self.__totalStudios

    # Setter and Getter for rating
    def setRating(self, rating): self.__rating = rating
    def getRating(self): return self.__rating